src/DependencyEmitter/BasicDependencyEmitter.php

```diff
@@ -33,6 +33,42 @@ public function supportsParser(AstParserInterface $astParser)
         return $astParser instanceof NikicPhpParser;
     }
 
+    public function applyDependencies(
```

Comment:
Because [symfony coding standard](https://symfony.com/doc/current/contributing/code/standards.html) : "Declare public methods first, then protected ones and finally private ones."

I find this rule really good and helpful! Big 👍